packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -261,7 +256,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 				if (entry.minTTLInSeconds - timeLapseSinceLocalUpload > entry.minTTLInSeconds / 2) {
 					this.pendingBlobs.set(localId, {
 						blob,
-						status: PendingBlobStatus.OfflinePendingOp,
+						status: PendingBlobStatus.SendingOp,
```

Comment:
how is the sending op status different from the opsent property?

Reply:
SendingOp status was changed by actually uploading the blob during onUploadResolve. opsent is true regardless the op was sent via onUploadResolve or stashing. I believe there should be a way to reduce concepts as well. I'll think about that after resolving other issues